TestHelper/Fixture.swift

```diff
@@ -73,7 +73,7 @@ public class Fixture: NSObject {
         return locations.map { CLLocation($0) }
     }
     
-    public class func route(from jsonFile: String) -> Route {
+    @objc public class func route(from jsonFile: String) -> Route {
```

Comment:
The Objective-C selector name should be `-routeFromJSONFilePath:`.

Reply:
File path would indicate a full path or URL, but it's really just a name, a fixture bundled in the TestHelper framework. So `-routeFromJSONFileName:` is the most idiomatic I could come up with.